packages/integration-objects/README.md

```diff
@@ -16,3 +16,49 @@ The following OData services are provided:
 
 > NOTE:
 To consume integration objects, the project must contain libraries relevant to [SAP Cloud Application Programming Model](https://cap.cloud.sap/docs/) or CAP.
+
+## Usage
+
+In reference application, a typical package.json to consume Integration Objects would be as below:
+``` {
+  "name": "integration-objects-refapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "postinstall": "cp -r node_modules/@sap-cloud-sdk/currency-conversion-integration-objects/srv/* srv/ && cp -r node_modules/@sap-cloud-sdk/currency-conversion-integration-objects/db/* db/ && cp -r node_modules/@sap-cloud-sdk/currency-conversion-integration-objects/cds-security.json .",
+    "build": "cds build/all"
+  },
+  "cds": {
+    "requires": {
+      "uaa": {
+        "kind": "xsuaa"
+      },
+      "db": {
+        "kind": "hana"
+      }
+    }
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@sap-cloud-sdk/currency-conversion-core": "0.0.1",
+    "@sap-cloud-sdk/currency-conversion-models": "0.0.1",
+    "@sap-cloud-sdk/currency-conversion-data-adapter": "0.0.1",
+    "@sap-cloud-sdk/currency-conversion-integration-objects": "0.0.1"
+  }
+}
+
+```
+The following steps need to be done beforehand: 
+1. Create *srv* and *db* folders in your project as a typical CDS project would look like.
+	
+2. Ensure that the *postinstall* hook in the package.json is present as above. The folders named *srv* and *db* into which you'd be copying the downloaded artifacts should be the same as that you created in the previous step. This step also copies the *cds-security.json* to the parent directory.
+	
+3. To run the APIs, the user would need to be assigned the roles *CurrencyConversionDisplay* or *CurrencyConversionConfigure* depending on what roles the user would need as the token generation would require this step to have been done in prior. The *xsuaa* section under the *cds* in the package.json takes care of this.
+	
+4. The DB kind would need to be specified based on the DB being used. The above example shows hana being used. This configuration would generate native artifacts. 
+
+5. To build this entire project, you'd need to run the *build* script from the package.json. This step generates all the corresponding native artifacts for *db* and *srv*. It generates the *manifest.yml* files too in case you'd need to deploy the modules individually.
```

Comment:
```suggestion
5. To build this entire project, you need to run the *build* script from the package.json. This step generates all the corresponding native artifacts for *db* and *srv*. It generates the *manifest.yml* files too in case you need to deploy the modules individually.
```